keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Tuning the number of domains in PodTopologySpread
+kep-number: 3022
+authors:
+  - "@sanposhiho"
+owning-sig: sig-scheduling
```

Comment:
Add `sig-autoscaling` as participating sig and @MaciekPytel as reviewer.

Reply:
Sure.